bin/portal-bridge/src/cli.rs

```diff
@@ -33,6 +32,11 @@ use crate::{
 pub const DEFAULT_SUBNETWORK: &str = "history";
 pub const DEFAULT_EXECUTABLE_PATH: &str = "./target/debug/trin";
 
+/// The maximum number of enrs to return in a response,
+/// limiting the number of OFFER requests spawned by the bridge
+/// for each piece of content
```

Comment:
```suggestion
/// The maximum number of peers to send each piece of content.
///
/// This is used as a parameter in census, which selects the peers.
```

nit: I think this comment was written mostly from census point of view, and now is a bit outdated. Feel free to update it.